src/snowflake/snowpark/dataframe.py

```diff
@@ -1904,6 +1933,12 @@ def join(
                 You can also use ``join_type`` keyword to specify this condition.
                 Note that to avoid breaking changes, currently when ``join_type`` is specified,
                 it overrides ``how``.
+            lsuffix: suffix the overlapping columns of the left DataFrame in the result DataFrame when the left and right DataFrames have overlapping column names.
```

Comment:
I feel the description in pandas doc is more concise https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.join.html

Reply:
@sfc-gh-mabrennan What do you think is better? This one and `Suffix to use from left frame’s overlapping columns.`.
Or you can suggest one better than both.

Reply:
How about:
Suffix to add to the overlapping columns of the left DataFrame.

But either one is probably fine. It's better to not copy the pandas docs verbatim though.